opm/parser/eclipse/Units/UnitSystem.cpp

```diff
@@ -129,6 +138,9 @@ namespace {
         1, /* gas-oil ratio */
```

Comment:
While not a part of this PR, is not this wrong (also oil-gas ratio and the reverse relations in from_field[]): it should be GasSurfaceVolume/LiquidSurfaceVolume (and vice versa in the right spots), that is MSCF/STB.